camayoc/constants.py

```diff
@@ -5,6 +5,9 @@
 MASKED_PASSWORD_OUTPUT = r"\*{8}"
 """Regex that matches password on outputs."""
 
+TOKEN_INPUT = "Provide( a)? token for OpenShift authentication.\r\nToken:"
```

Comment:
Is this space intentional? maybe you did want the following 

```suggestion
TOKEN_INPUT = "Provide (a)? token for OpenShift authentication.\r\nToken:"
```

Reply:
I think this string is a regex, in which case Ruda's version will match `Provide a token` and `Provide token`. Your version would match double space between `provide` and `token`.

However, that raises a good question - why do we need to match both in the first place? Can this string change between `qpc` invocations?

Reply:
It shouldn't. See https://github.com/quipucords/qpc/blob/master/qpc/messages.py#L299

But I honestly don't love this text has OpenShift specifically tied to it. It is possible to (try) adding a credential for another source with a `token`. It will fail, but that's not the point (the validation logic is on server side and qpc should be just a veeeeery thin layer).

Reply:
@ruda before merging can we update the expected input?
Link to the updated message: https://github.com/quipucords/qpc/blob/1.2.2/qpc/messages.py#L305

Reply:
Fixed in 673d87cf8122884d42f80a0f9044a076ff1ca509